cmd/libs/go2idl/openapi-gen/generators/openapi.go

```diff
@@ -40,15 +40,30 @@ const tagOptional = "optional"
 
 // Known values for the tag.
 const (
-	tagValueTrue       = "true"
-	tagValueFalse      = "false"
-	tagExtensionPrefix = "x-kubernetes-"
+	tagValueTrue              = "true"
+	tagValueFalse             = "false"
+	tagExtensionPrefix        = "x-kubernetes-"
+	tagPatchStrategy          = "patchStrategy"
+	tagPatchMergeKey          = "patchMergeKey"
+	patchStrategyExtensionKey = "x-kubernetes-patch-strategy"
```

Comment:
I was doing it that way, but because we had patch-strategy and patchStrategy both, I though it could be mixed up and I am not sure about the naming of only "patch-strategy". Something like patchStrategyPartialExtensionKey? :)